src/v2/Apps/Auctions/auctionsRoutes.tsx

```diff
@@ -5,34 +5,89 @@ const AuctionsApp = loadable(() => import("./AuctionsApp"), {
   resolveComponent: component => component.AuctionsAppFragmentContainer,
 })
 
+const CurrentAuctions = loadable(() => import("./Routes/CurrentAuctions"), {
+  resolveComponent: component => component.CurrentAuctionsFragmentContainer,
+})
+
+const UpcomingAuctions = loadable(() => import("./Routes/UpcomingAuctions"), {
+  resolveComponent: component => component.UpcomingAuctionsFragmentContainer,
+})
+
+const PastAuctions = loadable(() => import("./Routes/PastAuctions"), {
+  resolveComponent: component => component.PastAuctionsFragmentContainer,
+})
```

Comment:
This weird format for importing stuff is how we import individual components in our various routes files, which is how we do [code splitting](https://reactjs.org/docs/code-splitting.html). 

What this means in practice, is that when a user visits a particular route (say `/auctions/past`) they will _only_ download the relevant js for that route. When they navigate to a new route (say `/auctions/upcoming`), that will kick off a new request and download more js. At scale this keeps initial request time down by only downloading code thats actually being used on demand.